app/assets/stylesheets/common.scss

```diff
@@ -2304,6 +2304,11 @@ a.button {
     font-size: 13px;
     background: #e8e8e8;
     padding: 2px 3px;
+    white-space: pre-wrap;       /* css-3 */
+    white-space: -moz-pre-wrap;  /* Mozilla, since 1999 */
+    white-space: -pre-wrap;      /* Opera 4-6 */
+    white-space: -o-pre-wrap;    /* Opera 7 */
```

Comment:
Hello. I am trying to make my first open source contribution and was interested in finishing this so that it can be approved and merged in. 

It looks as if we remove the following lines:
`white-space: -moz-pre-wrap;  /* Mozilla, since 1999 */`
`white-space: -pre-wrap;          /* Opera 4-6 */`
`white-space: -o-pre-wrap;      /* Opera 7 */`
then the text-wrap still works. Were these 3 lines the ones that `autoprefixer-rails` would add?

If so, I can clone this branch and make the fix.

Reply:
@RoyallDesigns Thanks for the offer, and you're quite right that deleting those lines is what I meant, but it's probably easier if I just merge it myself with that change.

I'd definitely encourage you to find something else to work on though - it will be more fulfilling if you can find something where you can actually make a difference rather than just executing a predetermined edit like this though.